### PR TITLE
chore: updates example directory

### PR DESCRIPTION
**Pull request by tnolet**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

### Notes for the reviewer

Updates the examples directory:

- removes TS for now
- adds alert channel
- reshuffles examples to a use case

### CODE REVIEW

**Review by umutuzgur — DISMISSED** [review]
LGTM

File: examples/simple-project/_github/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# - Runs after deployment event
+# - Uses the target_url provided in the deployment event
+# - Dry run all the checks on Checkly
+# - If the dry run passes, deploy the checks.
+#
+#  You can of course trigger the Checkly CLI in other workflows or steps inside your GH Action configuration.
+name: checkly
+on: [deployment_status]
+env:
+  ENVIRONMENT_URL: ${{ github.event.deployment_status.target_url }}
```
Comment (umutuzgur) [inline examples/simple-project/_github/workflow.yml:13]: I don't think that is correct. We gotta pass check envs with the `-e` flag

Comment (tnolet) [reply examples/simple-project/_github/workflow.yml:13]: @umutuzgur not in my example code, it already works with https://github.com/checkly/checkly-cli/blob/a611222b2917a4414f75be49b5a25d2e623283b6/examples/simple-project/src/defaults.js

Comment (tnolet) [reply examples/simple-project/_github/workflow.yml:13]: ...or does it...

Comment (umutuzgur) [reply examples/simple-project/_github/workflow.yml:13]: I'm not sure if it works. I can guarantee it won't work with typescript given we have an in-memory transpiler in between

Comment (tnolet) [reply examples/simple-project/_github/workflow.yml:13]: I'm testing this right now

Comment (tnolet) [reply examples/simple-project/_github/workflow.yml:13]: @umutuzgur you were right, it's fixed now

**Review by umutuzgur — APPROVED** [review]
LGTM

**Pull request merged**